postgres-13/Dockerfile

```diff
@@ -1,6 +1,6 @@
 # This file is auto generated from it's template,
 # see citusdata/tools/packaging_automation/templates/docker/latest/latest.tmpl.dockerfile.
```

Comment:
1)

```suggestion
# see citusdata/tools/packaging_automation/templates/docker/postgres-13/postgres-13.tmpl.dockerfile.
```

2) Same probably applies to pg-12 file too

3) Also, we should rename pg14 file to pg13 in tools repo as well.

Reply:
Fixed it